source/jade/_mixins/user-mixins.jade

```diff
@@ -86,7 +86,7 @@ mixin pwd-reset-form
 mixin alert-login-failed
   if viewModel && viewModel.errorMessage
     .alert(role='alert')
-      #message= 'Error: ' + viewModel.errorMessage
+      #message!= 'Error: ' + viewModel.errorMessage
```

Comment:
Hey @michaelhidalgo I'm not happy with doing this since it opens up the potential to xss.

The reason you are doing the un-escape is because you want to put an email (i.e. html) inside a controller provided string, which is a pattern we need to avoid

In fact the script we are working at https://github.com/TeamMentor/TM_4_0_Design/issues/385 should be picking up cases like this (which means that this PR should had a failed travis result)